wpilibc/src/main/native/include/frc/controller/ProfiledPIDController.h

```diff
@@ -292,9 +293,31 @@ class ProfiledPIDController
   }
 
   /**
-   * Reset the previous error, the integral term, and disable the controller.
+   * Reset the previous error and the integral term.
+   * @param measurement the current measured State of the system.
    */
-  void Reset() { m_controller.Reset(); }
+  void Reset(State measurement) {
+    m_controller.Reset();
+    m_setpoint = measurement;
+  }
+
+  /**
+   * Reset the previous error and the integral term.
+   * @param measuredPosition the current measured position of the system.
+   * @param measuredVelocity the current measured velocity of the system.
+   */
+  void Reset(Distance_t measuredPosition, Velocity_t measuredVelocity) {
+    Reset(State{measuredPosition, measuredVelocity});
+  }
+
+  /**
+   * Reset the previous error and the integral term.
+   * @param measuredPosition the current measured position of the system. The
```

Comment:
There should be a blank line between the description and the `@param` tags. Furthermore, The first word in the `@param` description should be capitalized.

Reply:
addressed in b04b686